app/models/bot.rb

```diff
@@ -12,8 +12,37 @@ class Bot < ActiveRecord::Base
   validates :hair_color, presence: true
   validates :user_id, presence: true
 
+  WELCOME_MESSAGE = "Yeiii!"
+  UNKNOWN_COMMAND_MESSAGE = "This is a test"
+
   def avatar
-    gender + '-' + hair_color + '-' + eye_color + '.png'
+    gender + "-" + hair_color + "-" + eye_color + ".png"
+  end
+
+  def send_welcome_message
+    send_message(user.full_phone_number, WELCOME_MESSAGE)
   end
 
+  def send_message(recipient_phone,body)
```

Comment:
Unused method argument - `recipient_phone`. If it's necessary, use `_` or `_recipient_phone` as an argument name to indicate that it won't be used.<br>Space missing after comma.